udata_front/theme/gouvfr/assets/js/components/AdminTable/AdminDatasetsTable/AdminDatasetsTable.vue

```diff
@@ -0,0 +1,191 @@
+<template>
+  <AdminTable>
+    <thead>
+      <tr>
+        <AdminTableTh
+          :sortable="true"
+          @sort="(direction) => updateSort('title', direction)"
+          :sorted="sorted('title')"
+          scope="col"
+        >
+          {{ t('Title of the dataset') }}
+        </AdminTableTh>
+        <AdminTableTh scope="col">{{ t("Status") }}</AdminTableTh>
+        <AdminTableTh
+          :sortable="true"
+          @sort="(direction) => updateSort('created', direction)"
+          :sorted="sorted('created')"
+          scope="col"
+        >
+          {{ t('Created at') }}
+        </AdminTableTh>
+        <AdminTableTh
+          :sortable="true"
+          @sort="(direction) => updateSort('last_update', direction)"
+          :sorted="sorted('last_update')"
+          scope="col"
+        >
+          {{ t('Updated at') }}
+        </AdminTableTh>
+        <AdminTableTh scope="col">{{ t('Files') }}</AdminTableTh>
+        <AdminTableTh scope="col">{{ t('Score') }}</AdminTableTh>
+        <AdminTableTh scope="col">
+          <Tooltip>
+            <span class="fr-icon--sm fr-icon-chat-3-line" aria-hidden="true"></span>
+            <template #tooltip> {{ t('Discussions') }}</template>
+          </Tooltip>
+        </AdminTableTh>
+        <AdminTableTh
+          :sortable="true"
+          @sort="(direction) => updateSort('views', direction)"
+          :sorted="sorted('views')"
+          scope="col"
+        >
+          <Tooltip>
+            <span class="fr-icon--sm fr-icon-eye-line" aria-hidden="true"></span>
+            <template #tooltip> {{ t('Views') }}</template>
+          </Tooltip>
+        </AdminTableTh>
+        <AdminTableTh scope="col">
+          <Tooltip>
+            <span class="fr-icon--sm fr-icon-download-line" aria-hidden="true"></span>
+            <template #tooltip> {{ t('Downloads') }}</template>
+          </Tooltip>
+        </AdminTableTh>
+        <AdminTableTh
+          :sortable="true"
+          @sort="(direction) => updateSort('reuses', direction)"
+          :sorted="sorted('reuses')"
+          scope="col"
+        >
+          <Tooltip>
+            <span class="fr-icon--sm fr-icon-line-chart-line" aria-hidden="true"></span>
+            <template #tooltip> {{ t('Reuses') }}</template>
+          </Tooltip>
+        </AdminTableTh>
+        <AdminTableTh
+          :sortable="true"
+          @sort="(direction) => updateSort('followers', direction)"
+          :sorted="sorted('followers')"
+          scope="col"
+        >
+          <Tooltip>
+            <span class="fr-icon--sm fr-icon-star-s-line" aria-hidden="true"></span>
+            <template #tooltip> {{ t('Favorites') }}</template>
+          </Tooltip>
+        </AdminTableTh>
+      </tr>
+    </thead>
+    <tbody>
+      <tr v-for="dataset in datasets" :key="dataset.id">
+        <AdminTableTd>
+          <AdminContentWithTooltip>
```

Comment:
I'm not sure I've understood how AdminContentWithTooltip is used?

Reply:
It shows the cell content in a tooltip on smaller screen size